study1/week3/reduce-array-size-to-the-half.java

```diff
@@ -0,0 +1,30 @@
+// https://leetcode.com/problems/reduce-array-size-to-the-half/
+// Time Complexity: O(n), n: arr.length
```

Comment:
Can you try to thing once more? What do you think is the highest factor in the computation?